src/client/sandbox/event/simulator.js

```diff
@@ -610,6 +646,10 @@ export default class EventSimulator {
         return this._dispatchEvent(el, 'focus', false, this.DISPATCHED_EVENT_FLAG);
     }
 
+    storage (window, options) {
```

Comment:
I do not agree, all events should be public, let the testcafe has all set of tools